### PR TITLE
Add foreman_ansible_core 0.0.1 [rpm]

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by dLobatog** [comment]
Prereq for https://github.com/theforeman/foreman-packaging/pull/1335

File: rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_ansible_core
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?dist}
+Summary: Ansible integration with Foreman (theforeman.org): core bits
+Group: Development/Languages
+License: GPL-3
```
Comment (domcleal) [inline rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: GPLv3 (https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Software_License_List)

**Comment by dLobatog** [comment]
1. f0002ab2d3356da79fe9718cef3c9bbb15ae1b6c , thanks for the heads-up
2. The gemspec doesn't mark json_inventory.sh as executable therefore gem2rpm didn't find it. I'm not quite sure we really want that in the binpath, it's retrieved in the code via `File.expand_path('../../bin/json_inventory.sh', File.dirname(__FILE__))`
3. fixed, thanks

File: rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
```
Comment (dLobatog) [inline rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: I make json_inventory.sh executable here, but really no need to put it in /usr/bin as the gem calls it through 'File.expand_path('../../bin/json_inventory.sh''

**Comment by domcleal** [comment]
Merged and building, thanks @dLobatog.

**Pull request merged**